docs/learn/learn-polkadot-opengov.md

```diff
@@ -336,12 +412,19 @@ Decision Period.
 Once the proposal exits the Lead-in Period and enters the Voting Period, to be approved, it must
 satisfy the approval and support criteria for the **Confirmation Period**.
 
-- **Approval** is defined as the share of approval (_aye_ votes) vote-weight (after adjustment for
-  [conviction](#voluntary-locking)) against the total vote-weight (_aye_, _nay_, and _abstained_).
+- **Approval** is defined as the share of [conviction](#voluntary-locking)-weighted _aye_ votes
+  against the conviction-weighted total of _aye_ and _nay_ votes. The code implementation can be viewed
+  [here](https://github.com/paritytech/polkadot-sdk/blob/f2fbba3be1d7deaf7cfc731cea00552c212ddfcf/substrate/frame/conviction-voting/src/types.rs#L77)
 - **Support** is the total number of _aye_ and _abstain_ votes (ignoring any adjustment for
   conviction) compared to the total possible votes ([active issuance](learn-DOT.md#token-issuance))
   that could be made in the system. In case of _split_ votes, only _aye_ and _abstain_ will count.
 
+For example, let us consider a hypothetical example where the total active issuance is {{ polkadot: 100 DOT :polkadot }}{{ kusama: 100 KSM :kusama }}
+and an account A with 10 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "Aye" with 4x conviction, where as account B votes "Nay" with 5 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} with 2x conviction 
+and finally account C with 20 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "abstain". In this scenario, only 45 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} from the total active issuance participated in voting on the
+referendum. Now, let us calculate the Approval and Support values for that referendum. Approval is calculated as Aye'/ Aye' + Nay', where Aye' and Nay' are the votes after 
```

Comment:
```suggestion
referendum. Now, let us calculate the Approval and Support values for that referendum. Approval is calculated as (Aye) / (Aye + Nay)  where "Aye" and "Nay" are the votes after 
```

Reply:
Aye' is Aye with conviction. It is important to differentiate this variable from Aye which is in support formula, which corresponds to DOT used to vote Aye.